css-two-column-layout/styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+  width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#header,
+.smallheaderdiv {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.column1-8 {
+  flex-basis: calc(100%/8);
+}
+
+.column3-8 {
+  flex-basis: calc(300%/8);
+  margin-top: 2.5rem;
+}
+
+img {
+  scale: 75%;
+}
+
+#header {
+  font-size: 3rem;
+  font-weight: bold;
+  margin-bottom: 0;
+  margin-top: 2rem;
+}
+
+#descriptor {
+  text-align: center;
+  flex-basis: 78%;
+}
+
+#descriptor,
+.descriptor {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 100;
+}
+
+.smallheader,
+.descriptor {
+  font-size: 0.8rem;
+  flex-basis: 90%;
+  margin: 0.2rem;
+}
+
+.descriptordiv,
+.smallheaderdiv {
+  display: flex;
+}
+
+.learn {
+  background-color: white;
+  font-weight: bold;
+  color: #d55a53;
+  border: 1.5px solid #d55a53;
+  font-size: 0.7rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-right: 1.5rem;
+  /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
```

Comment:
Don't add `eslint` rules without asking me first! They are usually giving you an error for a reason. Reach out to me about the error, and I will provide insight